doc/user/content/sql/copy-from.md

```diff
@@ -6,7 +6,8 @@ menu:
         parent: "sql"
 ---
 
-`COPY FROM` copies data into a table using the [Postgres COPY protocol](https://www.postgresql.org/docs/current/sql-copy.html).
+`COPY FROM` copies data into a table using the [Postgres COPY
```

Comment:
This line and the one below were perhaps better as "one line per sentence" instead of wrapping.

Reply:
Sure thing